### PR TITLE
AM: add information in Expression Language section about the access t…

### PR DESCRIPTION
**Pull request by leleueri**
…o the PAR values

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://graviteedocs.blob.core.windows.net/am-47-make-par-values-accessible/index.html)

### CODE REVIEW

**Review by farmborough — APPROVED** [review]

File: pages/am/3.x/user-guide/expression-language/user-guide-expression-language.adoc [hunk 1]
```diff
@@ -164,6 +164,25 @@ This expression will return a UserProperties object with following attributes:
 
 NOTE: As you may have noticed, `claims` and `additionalInformation` contain the same data. Depending on the place where the Expression Language is defined, the `claims` attribute may not be accessible whereas the `additionalInformation` attribute will always be present.
 
+==== Context Attributes
+
+The context attributes will contains all the information you may add using Policies into the AM Flows. In addition of these information, you will find the `authFlow` attribute. This `authFlow` attribute is used to accumulate information accross the authentication flow thanks to the Enrich Authentication Flow Policy, it also contains the request parameters you defined using the link:https://datatracker.ietf.org/doc/html/rfc9126[PAR specification].
```
Comment (ivan-gravitee) [inline pages/am/3.x/user-guide/expression-language/user-guide-expression-language.adoc:169]:
```suggestion
The Context attributes contain all the information you can add to AM Flows using policies. 
* You can use the `authFlow` attribute to accumulate information across the authentication flow thanks to the Enrich Authentication Flow Policy. 
* The `authFlow.requestParameters` attribute contains the request parameters defined using the link:https://datatracker.ietf.org/doc/html/rfc9126[PAR specification].  
```

File: pages/am/3.x/user-guide/expression-language/user-guide-expression-language.adoc [hunk 2]
```diff
@@ -164,6 +164,25 @@ This expression will return a UserProperties object with following attributes:
 
 NOTE: As you may have noticed, `claims` and `additionalInformation` contain the same data. Depending on the place where the Expression Language is defined, the `claims` attribute may not be accessible whereas the `additionalInformation` attribute will always be present.
 
+==== Context Attributes
```
Comment (ivan-gravitee) [inline pages/am/3.x/user-guide/expression-language/user-guide-expression-language.adoc:167]:
```suggestion
==== Context attributes
```

**Review by ivan-gravitee — CHANGES_REQUESTED** [review]
Just a few small edits

**Review by ivan-gravitee — APPROVED** [review]
LGTM

**Pull request merged**